views/error.pug

```diff
@@ -4,6 +4,13 @@ block page_name
   | Error
 
 block content
-  p Sorry that did not work
-  p= message
-  a(href='javascript:history.back()').btn.btn-primary Go back
+  div.sp_blk
```

Comment:
These class names are not very readable, and also don't follow the existing snake-case convention. I would expand them to full words.

Reply:
ok